src/sentry/integrations/slack/message_builder/issues.py

```diff
@@ -367,6 +370,81 @@ def build(self) -> SlackBody:
         )
 
 
+class SlackReleaseIssuesMessageBuilder(SlackMessageBuilder):
+    """Same as SlackIssuesMessageBuilder but for new issues in a release"""
+
+    def __init__(
+        self,
+        group: Group,
+        event: Event | None = None,
+        tags: set[str] | None = None,
+        identity: Identity | None = None,
+        actions: Sequence[MessageAction] | None = None,
+        rules: list[Rule] | None = None,
+        link_to_event: bool = False,
+        issue_details: bool = False,
+        notification: ProjectNotification | None = None,
+        recipient: Team | User | None = None,
+        last_release: Release | None = None,
+    ) -> None:
+        super().__init__()
+        self.group = group
+        self.event = event
+        self.tags = tags
+        self.identity = identity
+        self.actions = actions
+        self.rules = rules
+        self.link_to_event = link_to_event
+        self.issue_details = issue_details
+        self.notification = notification
+        self.recipient = recipient
+        self.last_release = last_release
+
+    def build(self) -> SlackBody:
+        text = build_attachment_text(self.group, self.event) or ""
+        project = Project.objects.get_from_cache(id=self.group.project_id)
+
+        # If an event is unspecified, use the tags of the latest event (if one exists).
+        event_for_tags = self.event or self.group.get_latest_event()
+        color = get_color(event_for_tags, self.notification)
+        fields = build_tag_fields(event_for_tags, self.tags)
+        footer = (
+            self.notification.build_notification_footer(self.recipient)
+            if self.notification and self.recipient
+            else build_footer(self.group, project, self.rules)
+        )
+        obj = self.event if self.event is not None else self.group
+        if not self.issue_details or (self.recipient and isinstance(self.recipient, Team)):
+            payload_actions, text, color = build_actions(
+                self.group, project, text, color, self.actions, self.identity
+            )
+        else:
+            payload_actions = []
+
+        issue_title = build_attachment_title(obj)
+        title_url = get_title_link(
+            self.group, self.event, self.link_to_event, self.issue_details, self.notification
+        )
+        release = (
+            parse_release(self.last_release.version)["description"] if self.last_release else ""
+        )
+        release_url = absolute_uri(
+            f"/organizations/{self.group.organization.slug}/releases/{release}/?project={self.group.project_id}"
+        )
```

Comment:
Hmmmm, I think to build the full `release_url` you have to provide the un-parsed release.version:
https://github.com/getsentry/sentry/blob/5d4ed401271eb28300edbeda065ef05fad58c5b4/src/sentry/notifications/notifications/activity/release.py#L151-L153